AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreEndToEndTests.swift

```diff
@@ -15,6 +15,7 @@ import Combine
 @testable import AmplifyTestCommon
 @testable import AWSDataStoreCategoryPlugin
 
+// swiftlint:disable all
```

Comment:
should disable only what you need instead of disabling everything, for example if the file is too long then you can either create another file and extend the class `DataStoreEndToEndTests` 
```
// File name can be something like `DataStoreEndToEndTests+Restart.swift`
extension DataStoreEndToEndTests {
  func testStopStart() 
  func testClearStart()
 etc..
```

or be more specific abotu what we are disabling, ie 
```
// swiftlint:disable type_body_length
// swiftlint:disable file_length
// swiftlint:disable line_length
```